application/jspmyadmin/src/main/java/com/jspmyadmin/framework/web/utils/MessageReader.java

```diff
@@ -93,9 +93,13 @@ private static void _readFile(File file, String locale) throws IOException {
 			bufferedReader = new BufferedReader(inputStreamReader);
 			messageMap = new ConcurrentHashMap<String, String>();
 			while ((line = bufferedReader.readLine()) != null) {
-				if (line.contains(_equals)) {
-					data = line.split(_equals);
-					messageMap.put(data[0].trim(), data[1].trim());
+				try {
+					if (line.contains(_equals)) {
+						data = line.split(_equals);
+						messageMap.put(data[0].trim(), data[1].trim());
+					}
+				} catch (Exception e) {
+					e.printStackTrace();
```

Comment:
This changes make application logging about missing values in translated properties.